examples/forall_multi-reductions.cpp

```diff
@@ -27,9 +26,8 @@
  *
  */
 
-template < typename t_exec_policy, typename t_multi_reduce_policy >
-struct Backend
-{
+template <typename t_exec_policy, typename t_multi_reduce_policy>
+struct Backend {
```

Comment:
I prefer the opening curly brace for a class/struct definition to be on it's own line. What do others think?

Reply:
I prefer that way too.